indra/newview/llluamanager.cpp

```diff
@@ -375,16 +368,16 @@ void LLRequireResolver::findModule()
 
     // not already cached - prep error message just in case
     auto fail{
-        [L=L, path=mPathToResolve]()
+        [L=L, path=mPathToResolve.u8string()]()
```

Comment:
I'd not be surprised if on Windows this would let the viewer fall flat on it's face in mPathToResolve contains a unicode path. Like C:\Prögrämme\ (that's not a real thing).
Something to create a test case for I suggest

Reply:
Good point. It seems that using `utf8str_to_utf16str` should be enough to fix it.

Reply:
Okay, except (a) it needs to work on Posix as well as Windows and (b) [we should be actively moving away from `llutf16string`](https://github.com/secondlife/viewer/blob/lua-startup/indra/llcommon/llstring.h#L635-L645). I'll submit an alternative.

Reply:
> C:\Prögrämme\

Windows Heavy Metal Edition